plugins/modules/dcnm_vrf.py

```diff
@@ -1431,59 +1463,65 @@ def push_to_remote(self, is_rollback=False):
+                                    else:
+                                        vrflite_con['VRF_LITE_CONN'][0]['IF_NAME'] = ext_values['IF_NAME']
+
+                                    if ad_l['dot1q']:
+                                        vrflite_con['VRF_LITE_CONN'][0]['DOT1Q_ID'] = str(ad_l['dot1q'])
+                                    else:
+                                        vrflite_con['VRF_LITE_CONN'][0]['DOT1Q_ID'] = str(ext_values['DOT1Q_ID'])
+
+                                    if ad_l['ipv4_addr']:
+                                        vrflite_con['VRF_LITE_CONN'][0]['IP_MASK'] = ad_l['ipv4_addr']
+                                    else:
+                                        vrflite_con['VRF_LITE_CONN'][0]['IP_MASK'] = ext_values['IP_MASK']
+
+                                    if ad_l['neighbor_ipv4']:
+                                        vrflite_con['VRF_LITE_CONN'][0]['NEIGHBOR_IP'] = ad_l['neighbor_ipv4']
+                                    else:
+                                        vrflite_con['VRF_LITE_CONN'][0]['NEIGHBOR_IP'] = ext_values['NEIGHBOR_IP']
+
+                                    vrflite_con['VRF_LITE_CONN'][0]['NEIGHBOR_ASN'] = '65535'
+
+                                    if ad_l['ipv6_addr']:
+                                        vrflite_con['VRF_LITE_CONN'][0]['IPV6_MASK'] = ad_l['ipv6_addr']
+                                    else:
+                                        vrflite_con['VRF_LITE_CONN'][0]['IPV6_MASK'] = ext_values['IPV6_MASK']
+
+                                    if ad_l['neighbor_ipv6']:
+                                        vrflite_con['VRF_LITE_CONN'][0]['IPV6_NEIGHBOR'] = ad_l['neighbor_ipv6']
+                                    else:
+                                        vrflite_con['VRF_LITE_CONN'][0]['IPV6_NEIGHBOR'] = ext_values['IPV6_NEIGHBOR']
+
+                                    vrflite_con['VRF_LITE_CONN'][0]['AUTO_VRF_LITE_FLAG'] = 'false'
+                                    vrflite_con['VRF_LITE_CONN'][0]['PEER_VRF_NAME'] = ad_l['peer_vrf']
+                                    vrflite_con['VRF_LITE_CONN'][0]['VRF_LITE_JYTHON_TEMPLATE'] = 'Ext_VRF_Lite_Jython'
+                                    extension_values['VRF_LITE_CONN'] = json.dumps(vrflite_con)
+
+                                    ms_con = {}
+                                    ms_con['MULTISITE_CONN'] = []
+                                    extension_values['MULTISITE_CONN'] = json.dumps(ms_con)
+
+                                    v_a['extensionValues'] = json.dumps(extension_values).replace(' ', '')
+                                    v_a['instanceValues'] = "{\"loopbackId\":\"\",\"loopbackIpAddress\":\"\",\"loopbackIpV6Address\":\"\"}"
+                                    del v_a['vrf_lite']
+
+                        if ext_values == None:
+                            msg = 'There is no VRF LITE capable interface in switch {}'.format(attach['ip_address'])
```

Comment:
*nit:* I would reword the `msg` slightly to read:

`'There is no VRF LITE capable interface on this switch {}'`

Reply:
Changed as suggested